web2.0/src/Components/Historic/HistData.js

```diff
@@ -0,0 +1,110 @@
+            className={currentData?.windData[i].windDirection.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.windData[i].windSpeed.dataLabel}
+            weatherData={currentData?.windData[i].windSpeed.data}
+            extraData={currentData?.windData[i].windSpeed.unit}
+            className={currentData?.windData[i].windSpeed.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].waveHeight.dataLabel}
+            weatherData={currentData?.swellData[i].waveHeight.data}
+            extraData={currentData?.swellData[i].waveHeight.unit}
+            className={currentData?.swellData[i].waveHeight.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].swellHeight.dataLabel}
+            weatherData={currentData?.swellData[i].swellHeight.data}
+            extraData={currentData?.swellData[i].swellHeight.unit}
+            className={currentData?.swellData[i].swellHeight.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].swellPeriod.dataLabel}
+            weatherData={currentData?.swellData[i].swellPeriod.data}
+            extraData={currentData?.swellData[i].swellPeriod.unit}
+            className={currentData?.swellData[i].swellPeriod.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].windWaveHeight.dataLabel}
+            weatherData={currentData?.swellData[i].windWaveHeight.data}
+            extraData={currentData?.swellData[i].windWaveHeight.unit}
+            className={currentData?.swellData[i].windWaveHeight.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].windWavePeriod.dataLabel}
+            weatherData={currentData?.swellData[i].windWavePeriod.data}
+            extraData={currentData?.swellData[i].windWavePeriod.unit}
+            className={currentData?.swellData[i].windWavePeriod.className}
+          />
+          <WeatherCard
+            weatherDataLabel={'Swell Direction'}
+            weatherData={currentData?.swellData[i].swellDirection.data}
+            extraData={currentData?.swellData[i].swellDirection.unit}
+            className={currentData?.swellData[i].swellDirection.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].windWaveDirection.dataLabel}
+            weatherData={currentData?.swellData[i].windWaveDirection.data}
+            extraData={currentData?.swellData[i].windWaveDirection.unit}
+            className={currentData?.swellData[i].windWaveDirection.className}
+          />
+          <WeatherCard
+            weatherDataLabel={currentData?.swellData[i].steepness.dataLabel}
+            weatherData={
+              currentData?.swellData[i].steepness.data === 'N/A'
+                ? 'FLAT'
```

Comment:
See my earlier comment where I ask for clarification on whether 'N/A' === 'FLAT.